.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Report a bug in Bee
+about: Create a report to help us improve
+title: ""
+labels: c-bug
+---
+
```

Comment:
This is a HackMD header. Is this by accident?

Reply:
It's also the metadata format for GH :)

Reply:
haha, yea :smile: